go.mod

```diff
@@ -1,14 +1,44 @@
 module fyne.io/x/fyne
 
-go 1.14
+go 1.20
```

Comment:
Can you not keep the Go module mostly as it was? We should not have 1.20 as the minimum version. Just include any new dependencies you need for this change so we can keep the changes minimal.